### PR TITLE
Add bqplot recipe

### PR DESCRIPTION
**Pull request by SylvainCorlay**
cc @jasongrout

### CODE REVIEW

File: recipes/bqplot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    number: 1
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+        - ipywidgets >=5.1.3
```
Comment (jakirkham) [inline recipes/bqplot/meta.yaml]: Based on the build failures, it seems like `numpy` is a dependency here.

File: recipes/bqplot/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    md5: e0ab4e1f478b85c59e774eb58317cd67
+
+build:
+    number: 1
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
```
Comment (jakirkham) [inline recipes/bqplot/meta.yaml]: Is `numpy` also a build dependency? Are we linking against it here?

Comment (jakirkham) [inline recipes/bqplot/meta.yaml]: I looked closer and it seems the answer is that it is a dependency, but we are not linking to it. So, this is fine.

File: recipes/bqplot/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    name: bqplot
+    version: {{ version }}
+
+source:
+    fn: bqplot-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/b/bqplot/bqplot-{{ version }}.tar.gz
+    md5: e0ab4e1f478b85c59e774eb58317cd67
+
+build:
+    number: 1
```
Comment (jakirkham) [inline recipes/bqplot/meta.yaml]: Could we please reset the build number to `0`?

File: recipes/bqplot/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+about:
+    home: https://github.com/bloomberg/bqplot
+    license: Apache License 2.0
+    summary: Plotting library for Jupyter
+
+extra:
+  recipe-maintainers:
+    - SylvainCorlay
+    - jasongrout
```
Comment (jakirkham) [inline recipes/bqplot/meta.yaml]: I'm guessing that you guys talked about this, but figured I should follow up.

Are you ok with being a maintainer of `bqplot`, @jasongrout?

Comment (jasongrout) [inline recipes/bqplot/meta.yaml]: Yep, I'm okay with it. Thanks for asking.

Comment (jakirkham) [inline recipes/bqplot/meta.yaml]: Awesome. Thanks for confirming.

**Comment by jakirkham** [comment]
So, there is one nit and this should be good to go after it is addressed.

**Comment by SylvainCorlay** [comment]
Done!

**Comment by jakirkham** [comment]
Awesome. Thanks for the addition, @SylvainCorlay.

**Comment by jakirkham** [comment]
Weird. AppVeyor got kind of messed up, but it had nothing to do with this.

Restarting. Though not expecting any surprises here.

**Comment by SylvainCorlay** [comment]
Cool! I will add ipyleaflet and pythreejs.

**Comment by jakirkham** [comment]
Green. 😄

**Pull request merged**